DependencyInjection/AbstractSmtpTransportConfigurator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SwiftmailerBundle\DependencyInjection;
+
+use Symfony\Component\Routing\RequestContext;
+
+/**
+ * Service configurator.
+ */
+class AbstractSmtpTransportConfigurator
+{
+    /**
+     * @var string
+     */
```

Comment:
Fixed. But why? The coding guidelines on http://symfony.com/doc/current/contributing/code/standards.html does not go into much detail about PHPdoc, but it does include an example of a @var used together with a private variable.